js/plates-model/crust.ts

```diff
@@ -104,10 +104,17 @@ export default class Crust {
     return result;
   }
 
-  get topRockType() {
-    if (!config.sedimentsInPlanetView) {
+  get sedimentThickness() {
+    if (isSediment(this.rockLayers[0].rock)) {
```

Comment:
Maybe document why you are only checking index 0 here?  The other getters seem to check all the rock layers.

Reply:
Done in the recently pushed commits.